lms/resources/_js_config/__init__.py

```diff
@@ -139,23 +139,25 @@ def enable_oauth2_redirect_error_mode(
                 "OAuth2RedirectError": {
                     "authUrl": auth_url,
                     "errorCode": error_code,
-                    "errorDetails": error_details,
                     "canvasScopes": canvas_scopes or [],
                 },
             }
         )
 
-    def enable_error_dialog_mode(self, error_code, error_details=""):
+        if error_details:
+            self._config["OAuth2RedirectError"]["errorDetails"] = error_details
```

Comment:
It's desirable for the frontend that `errorDetails` be omitted rather than `null`